.cicd/Jenkinsfile

```diff
@@ -149,7 +149,7 @@ pipeline {
                     stage('Functional WorkflowTaskTests') {
                         steps {
                             echo "Running simple workflow script task tests on ${env.SRW_PLATFORM} (using ${env.WORKSPACE})"
-                            sh 'bash --login "${WORKSPACE}/.cicd/scripts/srw_ftest.sh"'
+                            sh 'bash --login "${WORKSPACE}/.cicd/scripts/wrapper_srw_ftest.sh"'
```

Comment:
@EdwardSnyder-NOAA -

It might be possible to add a sleep command after line 152, like:

`sleep(time:15,unit:"MINUTES")`

The `Functional Workflow Task Tests` won't move on until after 15 minutes have passed.  Hopefully this would allow the tests to complete before moving on to the `Tests` stage.

Reply:
Thanks @MichaelLueken for the suggestion. I was thinking of adding logic that will check for the batch id and terminate once the job is done. Other thing I noticed is that if we add either of these changes it won't report if the wrapper scripts passed or not. So we will need to add code to display the results of the srw_ftest script. On a positive note, the srw_ftest test passed on Gaea, Jet, and Orion.

Reply:
Very nice, @EdwardSnyder-NOAA!  You make a very good point, it would probably be best to attempt your change first (using the sleep command won't guarantee that the job has passed, especially on machines like Jet, where the EPIC account is already in windfall).

One idea would be to grep for `FAIL` in the `functional_test_results_hera_intel.txt` text file.  If there is a fail, then you can return with a non-zero return code, leading to a failure for the stage.